include/IECore/LRUCache.h

```diff
@@ -148,6 +148,10 @@ class LRUCache : private boost::noncopyable
 		GetterFunction m_getter;
 		RemovalCallback m_removalCallback;
 
+#ifdef _MSC_VER
+	// Workaround for MSVC
+	public:
+#endif
```

Comment:
Could you comment on why this was needed?  I can't spot where this class is needed publicly.

Reply:
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2248	'IECore::LRUCache<IECore::MurmurHash,IECore::MurmurHash,IECore::LRUCachePolicy::Parallel,Key>::CacheEntry': cannot access private struct declared in class 'IECore::LRUCache<IECore::MurmurHash,IECore::MurmurHash,IECore::LRUCachePolicy::Parallel,Key>'	IECore	C:\Users\Alex\dev\cortex\include\IECore\LRUCache.inl	281	
Error	C2248	'IECore::LRUCache<IECore::MurmurHash,IECore::ConstObjectPtr,IECore::LRUCachePolicy::Parallel,Key>::CacheEntry': cannot access private struct declared in class 'IECore::LRUCache<IECore::MurmurHash,IECore::ConstObjectPtr,IECore::LRUCachePolicy::Parallel,Key>'	IECore	C:\Users\Alex\dev\cortex\include\IECore\LRUCache.inl	281